README.md

```diff
@@ -1,16 +1,16 @@
-# u8timeseries: Easy manipulation and forecasting of time series
+# darts: Easy manipulation and forecasting of time series
 
-![u8timeseries](https://github.com/unit8co/u8timeseries/raw/develop/static/images/logo-72dpi.png "u8timeseries") 
+![darts](https://github.com/unit8co/darts/raw/develop/static/images/logo-72dpi.png "darts") 
```

Comment:
We will need to change repository to show image

Reply:
yes, will do after merging